frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.unit.spec.tsx

```diff
@@ -43,18 +43,16 @@ const makeParameter = ({
 type SetupOpts = {
   parameters: WritebackParameter[];
   formSettings: ActionFormSettings;
-  isPublic?: boolean;
 };
 
-const setup = ({ parameters, formSettings, isPublic = false }: SetupOpts) => {
```

Comment:
in this test I dropped `isPublic` property and describe.each for it as it's not longer needed